src/OpenLoco/Map/BuildingTile.cpp

```diff
@@ -55,6 +55,25 @@ namespace OpenLoco::Map
         return town;
     }
 
+    // 0x0046902E
+    void sub_46902E(const Pos2& pos)
+    {
+        auto tile = TileManager::get(pos);
+        auto* surface = tile.surface();
+        if (surface != nullptr)
+        {
+            if (surface->hasHighTypeFlag())
+            {
+                surface->setHighTypeFlag(false);
+                surface->setVar6SLR5(0);
+                surface->setIndustry(IndustryId(0));
+                auto z = surface->baseZ() * 4;
+                Ui::ViewportManager::invalidate(pos, z, z + 32, ZoomLevel::eighth);
+                sub_497DC1(pos, 0, 0, 0xFFE2, 0);
```

Comment:
Will be -32 and not 0xFFE2. This suggests that sub_497DC1 should have a int16_t for rating.

Reply:
Can you explain how do you know it should be -32?

Reply:
0xFFE2 is a [two's complement](https://en.wikipedia.org/wiki/Two%27s_complement) representation of -32. You can switch between representations in IDA as well, fwiw.

Reply:
Oh I completely forgot about it, it's been a while since I worked with binary numbers. But isn't 0xFFE2 actually -30?

Reply:
You are quite right. I somehow dropped the 2 when calculating.